pkgs/tools/misc/fdupes/default.nix

```diff
@@ -1,17 +1,22 @@
-{ stdenv, fetchFromGitHub }:
+{ stdenv, fetchFromGitHub, autoconf, automake, ncurses, pcre2 }:
 
 stdenv.mkDerivation rec {
   pname = "fdupes";
-  version = "1.6.1";
+  version = "2.1.1";
 
   src = fetchFromGitHub {
     owner = "adrianlopezroche";
     repo  = "fdupes";
     rev   = "v${version}";
-    sha256 = "19b6vqblddaw8ccw4sn0qsqzbswlhrz8ia6n4m3hymvcxn8skpz9";
+    sha256 = "1c5hv7vkfxsii1qafhsynzp9zkwim47xkpk27sy64qdsjnhysdak";
   };
 
-  makeFlags = [ "PREFIX=$(out)" ];
+  nativeBuildInputs = [ autoconf automake ncurses pcre2 ];
+
+  configurePhase = ''
+    autoreconf --install
+    ./configure --prefix=${placeholder "out"}
+  '';
```

Comment:
Did you try if `autoreconfHook` works instead?

Reply:
No, thanks for pointing it out... I'm learning something each package I update.